commons-module/src/test/java/ru/qatools/school/webtests/WeatherWebTest.java

```diff
@@ -23,10 +23,27 @@ public void initSteps() {
     }
 
     @Test
-    @Title("Должны видеть виджет на главной странице")
+    @Title("Widget on main page must be seen")
     public void shouldSeeWidgetOnMainPage() {
         defaultSteps.openMainPageWithCity(MOSCOW);
-        defaultSteps.shouldSee(onMainPage().getWeatherWidget().get(0));
+        defaultSteps.shouldBeSeen(onMainPage().getWeatherWidgets().get(0));
+    }
+
+    @Test
+    @Title("City from widget must match city from URL")
```

Comment:
It's dangerous to go alone, take [this](https://en.wikipedia.org/wiki/Behavior-driven_development)!